libs/core/langchain_core/pydantic_v1/__init__.py

```diff
@@ -21,3 +22,25 @@
     _PYDANTIC_MAJOR_VERSION: int = int(metadata.version("pydantic").split(".")[0])
 except metadata.PackageNotFoundError:
     _PYDANTIC_MAJOR_VERSION = 0
+
+
+def _issubclass_base_model(type_: Type) -> bool:
```

Comment:
This shouldn't be in `pydantic_v1` -> can we place in langchain_core.utils.pydantic? 

pydantic_v1 is only the layer that checks that we're using the v1 namespace -- it's for the most part unaware of BaseModel from v2

Reply:
https://github.com/langchain-ai/langchain/pull/24418/files#diff-ff28020c5f1073a8b63bcd9d8b756a187fd682cb81935295120c63b207071896R23